content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/cpp.md

```diff
@@ -13,7 +13,7 @@ code_lang: cpp
 type: multi-code-lang
 code_lang_weight: 50
 further_reading:
-- link: "https://github.com/DataDog/dd-opentracing-cpp"
```

Comment:
Should we add a note to the top of this repo letting folks know to use dd-trace-cpp instead?

Reply:
Good idea. First step is I add the note at the top and soon we should deprecate the repo. What do you think?

Reply:
Assuming you mean archive/read-only, then 100% agreed